experiments/pages/childNormalPageConverter/childNormalPageConverter_SkuidPlatform.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "childNormalPageConverter_SkuidPlatform",
+	"apiVersion": "v1",
+	"composerSettings": {
+		"skuid.model(getMasterPageXML)": {
```

Comment:
@itmightbecody you should remove this, it will only work in your site

Reply:
Whoops.